install-dev/classes/controllerHttp.php

```diff
@@ -330,142 +336,129 @@ public function isFirstStep()
      */
     public function isLastStep()
     {
-        return self::$steps->isLastStep();
+        return self::getSteps()->isLastStep();
     }
 
     /**
      * Check is given step is already finished
      *
      * @param string $step
+     *
      * @return bool
      */
-    public function isStepFinished($step)
+    public function isStepFinished(string $step): bool
     {
-        return self::getStepOffset($step) < self::$steps->getOffset();
+        return $this->getStepOffset($step) < self::getSteps()->getOffset();
     }
 
     /**
-     * Get telephone used for this language
+     * Send AJAX response in JSON format {success: bool, message: string}
      *
-     * @return string
+     * @param bool $success
+     * @param string $message
      */
-    public function getPhone()
+    public function ajaxJsonAnswer($success, $message = '')
```

Comment:
```suggestion
    public function ajaxJsonAnswer(bool $success, string $message = '')
```

Reply:
I did not change this part, I'm afraid of side effect :sweat_smile:

Reply:
But I apply the fix, YOLO